website_sale_product_image_sample/readme/ROADMAP.rst

```diff
@@ -0,0 +1,5 @@
+* For the correct functioning of the module I had to rewrite .css_attribute_color, impacting the way of representing the selection by colors.
```

Comment:
What about adding an extra class that overrides those styles only for the image thumbnails?